AzureBot/UX/AzureBot.UX.Guidelines.md

```diff
@@ -0,0 +1,26 @@
+﻿# AzureBot UX Guidelines
```

Comment:
Let's create a new folder called "docs" and put this guide in there. 
Also let's just call it UX.Guidelines.md - since AzureBot is inferred.